pom.xml

```diff
@@ -68,6 +68,12 @@
             <artifactId>aws-lambda-java-events</artifactId>
             <version>3.4.0</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <artifactId>joda-time</artifactId>
+                    <groupId>joda-time</groupId>
+                </exclusion>
+            </exclusions>
```

Comment:
All these exclusions were due to multiple dependencies each including different versions of child libraries as dependencies.  With these changes, there are no longer any version conflicts.